data/brands/shop/convenience.json

```diff
@@ -1674,6 +1674,20 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Kuups",
+      "locationSet":{"include":["es"]},
+      "matchNames":["Kups"],
+      "matchTags":["shop/supermarket"]
+      "tags":{
+        "brand":"Kuups",
+        "name": "Kuups",
+        "operator": "Kuups Design International SL",
+        "operator:wikidata": "Q113464694",
```

Comment:
This entry is in brand tree, so you must add a brand:wikidata for Kuups or it won't be suggested.